tasks/tempest.yml

```diff
@@ -1,8 +1,25 @@
 ---
-- name: Get image
-  get_url:
+- name: Get image locally
+  become: false
+  local_action:
+    module: get_url
     url: "http://download.cirros-cloud.net/0.3.5/cirros-0.3.5-x86_64-disk.img"
     dest: "/tmp/cirros-0.3.5-x86_64-disk.img"
+    use_proxy: no
+
+- name: Convert image to raw format
+  become: false
+  local_action:
+    module: command
+    cmd: "qemu-img convert -f qcow2 -O raw /tmp/cirros-0.3.5-x86_64-disk.img /tmp/cirros-0.3.5-x86_64-disk.img.raw"
+
+- name: Upload image to rally server
```

Comment:
It is not clear why you should upload it to rally server. The image should be already stored in /tmp/ at rally server.

Reply:
I see now. The idea to upload and covert the image locally before uploading it into rally server. You could upload and convert at rally. After the conversion, you can delete the qcow2 format. This will reduce the download and upload of the images. However, if you think that this will not be used often, so you can omit my comment.

Reply:
As discussed before
- this approach avoids the need of installing qemu-img on rally server
- the raw image obtained is still quite small, so the upload should not take too much time
- this is not an operation we perform very often
So I would vote for keeping it as-is

Reply:
Ok. We can keep it as its as we do not expect to use the code often.